platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -605,6 +605,21 @@ fn build_solana_package(
             .iter()
             .filter_map(|target| {
                 if target.crate_types.contains(&"cdylib".to_string()) {
+                    let other_crate = if target.crate_types.contains(&"rlib".to_string()) {
```

Comment:
nit variable name:
```suggestion
                    let other_crate_type = if target.crate_types.contains(&"rlib".to_string()) {
```

Reply:
Fixed in https://github.com/anza-xyz/agave/pull/5729/commits/9b15d3db9df89aea67b9bb25007b7217a8c6d663